tests/conftest.py

```diff
@@ -8,16 +8,16 @@
 from pytest import fixture  # type: ignore
 from sympy import Expr
 
-from qadence import BackendName, DiffMode
-from qadence.blocks import AbstractBlock, chain, kron
-from qadence.blocks.utils import unroll_block_with_scaling
+from qadence.blocks.abstract import AbstractBlock
+from qadence.blocks.utils import chain, kron, unroll_block_with_scaling
```

Comment:
`operations` ?

Reply:
agreed, but lets discuss/decide on this in a separate MR/issue